include/caffeine/Interpreter/ExprEval.h

```diff
@@ -82,7 +83,7 @@ class ExprEvaluator : public llvm::InstVisitor<ExprEvaluator, LLVMValue> {
   LLVMValue visitFunction(llvm::Function& func);
 
   // Extras
-  OpRef visitGlobalData(llvm::Constant& cnst, unsigned AS);
+  void visitGlobalData(llvm::Constant& cnst, Pointer& ptr, unsigned AS);
```

Comment:
Passing the allocation directly makes this cleaner
```suggestion
  void visitGlobalData(llvm::Constant& cnst, Allocation& alloc, unsigned AS);
```